app/src/androidTest/java/com/github/h3lp3rs/h3lp/signin/NewUserSignInTest.kt

```diff
@@ -42,6 +47,8 @@ class NewUserSignInTest {
     fun setUp() {
```

Comment:
Since setup() is really similar in each test files, wouldn't be a good idea to put it in a utils class in test folder?

Reply:
Great remark! I thought about it but didn't realize until AFTER all the changes how many times this appeared. Should be changed at another refactoring, I already spent too much time on this one.